english/boring/boring.md

```diff
@@ -0,0 +1,3 @@
+# My Boring Story
+I may like to start by stating that I don't have what to say, but I actually do.
+Let's state a day in my  life at the summer. I woke up this morning and continued the Git course on udacity. My brother entered the room and asked me to go with him to repair the door at the other house. End of story
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.